com.unity.visualeffectgraph/Documentation~/VisualEffectGraphWindow.md

```diff
@@ -16,34 +16,35 @@ To open the Visual Effect Graph window, you can use any of the following methods
 | --------------------- | ------------------------------------------------------------ |
-| **Auto**              | **Toggle** : Toggles Auto-compilation of the Visual Effect Graph. |
-| **Compile**           | **Action** : Recompiles the currently opened Visual Effect Graph. |
-| **Save**              | **Action** : Save the currently opened Visual Effect Graph and its subgraphs. |
-| **Show in Project**   | **Action** : Focuses the currently opened Visual Effect Graph's Asset in the Project window. |
-| **Blackboard**        | **Toggle** : Toggles visibility of the **Blackboard Panel**. |
-| **Target GameObject** | **Toggle** : Toggles visibility of the **Target VisualEffect GameObject Panel**. |
-| **Advanced**          | **Menu:** Displays Advanced Properties. The options are <br/>&#8226; **Runtime Mode (Forced)**: Forces optimized compilation, even when the editor is open.<br/>&#8226; **Shader Validation (Forced)**: Performs a forced Shader compilation when the effect recompiles, even if no visual effect is visible. This displays the Shader errors in the Scene. <br/>&#8226; **Refresh UI**: Refreshes the window UI. |
+| **Save** <br/> ![](Images/save.png)             | **Action** : Use this to save the open Visual Effect Graph that and its subgraphs.<br/>**Dropdown**:<br/><br/>&#8226; **Save as…**: Saves the Visual Effect Graph under a specified name and/or location.<br/>&#8226; **Show in Inspector**: Focuses the Visual Effect Graph's Asset in the Inspector.|
+| **Compile** <br/> ![](Images/compile.png)             | **Action** : Recompiles the open Visual Effect Graph.<br/>**Dropdown**:<br/><br/>&#8226; **Auto Compile**: Toggles Auto-compilation of the Visual Effect Graph.<br/>&#8226; **Runtime Mode**: Forces optimized compilation, even when the editor is open.<br/>&#8226; **Shader Validation**: ForcesShader compilation when the effect recompiles, even if no visual effect is visible. This displays the Shader errors in the Scene.|
+| **Auto Attach** <br/> ![](Images/auto-attach.png)             | **Toggle**: Toggles the visibility of the Auto Attachment panel. The **Auto Attachment** panel allows you to attach the open Visual Effect Graph to a GameObject by selecting it in the Hierarchy. Once you have attached the visual effect to a GameObject, it enables the Visual Effect controls in the VFX Control panel and allows you to tweak gizmos in the Scene View.|
+| **Lock**           | **Toggle**: Toggles lock/unlock for auto attachments. If you set the toggle to unlocked - you can attach the open Visual Effect Graphto a GameObject by selecting items in the Hierarchy. If you set the toggle to locked - The GameObject that is currently attached becomes locked and auto attachments are disabled. The Visual Effect Graph then can not be attached by selecting items in the Hierarchy. You can manually keep the lock and change the attachment in the object picker of the Auto Attach panel.|
```

Comment:
Typo: "Graphto"

Reply:
should be fixed now

Reply:
I think I see typo too: 'ForcesShader'. We are missing a space, and I am unsure whether shader has to be capitalized.

Reply:
Fixed the typo. I think Shader is supposed to be capitalised because it is consistently capitalised throughout.